app/build.gradle.kts

```diff
@@ -29,6 +31,10 @@ android {
                 getDefaultProguardFile("proguard-android-optimize.txt"),
                 "proguard-rules.pro"
             )
+            buildConfigField("String", "BASE_URL", "\"https://tasky.pl-coding.com/\"")
+        }
+        debug {
+            buildConfigField("String", "BASE_URL", "\"https://tasky.pl-coding.com/\"")
         }
```

Comment:
If I dont add this I cannot use the BASE_URL String when working in debug variant

Reply:
Ohhh that's because you added it on release

You would just add it on the default config:
![image](https://github.com/MelvinKucuk/tasky/assets/16141845/6db860ed-8803-4324-89f0-ca12df75dd45)

Once you do that, you can remove it from both debug and release